src/plugin.json

```diff
@@ -1,49 +1,39 @@
 {
-  "type": "datasource",
-  "name": "Akenza Core Connector",
-  "id": "akenza-grafana-connector",
-  "metrics": true,
-  "info": {
-    "description": "Enables the Akenza Core as Data Source",
-    "author": {
-      "name": "David Giger",
-      "url": "https://akenza.com"
+    "type": "datasource",
+    "name": "Akenza Core Connector",
+    "id": "akenza-grafana-connector",
+    "metrics": true,
+    "info": {
+        "description": "Enables the Akenza Core as Data Source",
+        "author": {
+            "name": "David Giger",
+            "url": "https://akenza.com"
+        },
+        "keywords": [
+            "akenza core",
+            "akenza",
+            "IoT"
+        ],
+        "logos": {
+            "small": "img/akenza_small.svg",
+            "large": "img/akenza_large.svg"
+        },
+        "links": [
+            {
+                "name": "Website",
+                "url": "https://github.com/akenza-io/grafana-connector"
+            },
+            {
+                "name": "License",
+                "url": "https://github.com/akenza-io/grafana-connector/blob/master/LICENSE"
+            }
+        ],
+        "screenshots": [],
+        "version": "%VERSION%",
+        "updated": "%TODAY%"
     },
-    "keywords": [
-      "akenza core",
-      "akenza",
-      "IoT"
-    ],
-    "logos": {
-      "small": "img/logo.png",
-      "large": "img/logo.png"
-    },
-    "links": [
-      {
-        "name": "Website",
-        "url": "https://github.com/akenza-io/grafana-connector"
-      },
-      {
-        "name": "License",
-        "url": "https://github.com/akenza-io/grafana-connector/blob/master/LICENSE"
-      }
-    ],
-    "screenshots": [],
-    "version": "%VERSION%",
-    "updated": "%TODAY%"
-  },
-  "routes": [
-    {
-      "path": "connection-test",
-      "method": "*",
-      "url": "{{.JsonData.baseUrl}}/v1/environments",
-      "headers": [
-        { "name": "Api-Key", "content": "{{.SecureJsonData.apiKey}}" }
-      ]
+    "dependencies": {
+        "grafanaVersion": "7.x",
```

Comment:
is it still required to use version 7.x even if you are using legacy forms?

Reply:
Apparently 🤔
It's straight from their toolkit used to generate the structure of the plugin and that basically was their example of it 😉🤷🏻‍♂️

Reply:
And: all other elements are not compatible with older versions (just try running it on an older instance in docker-compose)...

Reply:
ok, then we go with this version